RELEASE_PROCESS.md

```diff
@@ -0,0 +1,268 @@
+          changes.
+        * Do not include any PRs that fix bugs which we introduced due to
+          new features/enhancements.  In other words, if it was working, broke, then
+          got fixed, there's no need to mention those items.
+
+   1. Commit and **sign** the `RELEASE_NOTES.md` changes, using the description
+      `Create release notes for vX.Y.Z` (where `X`, `Y`, and `Z` are the
+      actual version numbers).
+   1. Push your working branch to your github fork and create a new pull request.
+
+      * ***Ensure*** you properly select the base branch if not *master*.
+        For example, `vX.y.Z`.
+      * ***Before submitting*** the new PR, update the title with the
+        prefix `[CI:DOCS]` to avoid triggering lengthy automated testing.
+
+   1. If this is a release on a pre-existing version-named branch
+      (e.x. *release-candidate* or `-rhel`), open another PR against
+      the upstream *master* branch.  This is needed to ensure the new
+      notes are present for future releases.
+
+
+1. Create a new upstream release branch (if none already exist).
+
+   1. After the release-notes pull requests have merged, a release branch is
+      needed.  Branching ensures all changes are curated before inclusion in the
+      release, and no new features land after the *release-candidate* phases
+      are complete.
+   1. Ensure your local clone is fully up to date with the remote upstream
+      (`git remote update`).  Switch to this branch (`git checkout upstream/master`).
+   1. Make a new local branch for the release based on *master*.  For example,
+      `git checkout -b vX.Y`.  Where `X.Y` represent the complete release
+      version-name, including any suffix (if any) like `-rhel`.  ***DO NOT***
+      include any `-rc` suffix in the branch name.
+   1. Edit the `.cirrus.yml` file, changing the `DEST_BRANCH` value (under the
+      `env` section) to the new, complete branch name (e.x. `vX.Y`).
+       Commit and sign, using the description
+      `Cirrus: Update operating branch`.
+   1. Push the new branch otherwise unmodified (`git push upstream vX.Y`).
+   1. Automation will begin executing on the branch immediately.  Because
+      the repository allows out-of-sequence PR merging, it is possible that
+      merge order introduced bugs/defects.  To establish a clean
+      baseline, observe the initial CI run on the branch for any unexpected
+      failures.  This can be done by going directly to
+      `https://cirrus-ci.com/github/containers/podman/vX.Y`
+   1. If there are CI test or automation boops that need fixing on the branch,
+      attend to them using normal PR process (to *master* first, then backport
+      changes to the new branch).  Ideally, CI should be "green" on the new
+      branch before proceeding.
+
+1. Create a new branch-verification Cirrus-Cron entry.
+
+  1. This is to ensure CI's VM image timestamps are refreshed.  Without this,
```

Comment:
Misalignment of bullet point by a single space :) It's causing it to not be sub points
```suggestion
   1. This is to ensure CI's VM image timestamps are refreshed.  Without this,
```